### PR TITLE
Adding schema for ServiceBus 

### PR DESCRIPTION
**Pull request by sazeesha**

### CODE REVIEW

File: schemas/2014-04-01-preview/deploymentTemplate.json [hunk 1]
```diff
@@ -1,469 +1,476 @@
...
-    "contentVersion": {
-      "type": "string",
-      "pattern": "(^[0-9]+\\.[0-9]+\\.[0-9]+\\.[0-9]+$)",
-      "description": "A 4 number format for the version number of this template file. For example, 1.0.0.0"
-    },
-    "variables": {
-      "type": "object",
-      "description": "Variable definitions"
-    },
-    "parameters": {
```
Comment (wenwu449) [inline schemas/2014-04-01-preview/deploymentTemplate.json]: please revert this part change to keep the file history consistent.

File: schemas/2014-04-01-preview/deploymentTemplate.json [hunk 2]
```diff
@@ -1,469 +1,476 @@
...
-                      "$ref": "#/definitions/parametersLink"
-                    }
-                  }
-                },
-                {
-                  "type": "object",
-                  "properties": {
-                    "parameters": {
-                      "type": "object",
-                      "additionalProperties": {
```
Comment (wenwu449) [inline schemas/2014-04-01-preview/deploymentTemplate.json]: please revert this part change to keep the file history consistent.

File: schemas/2015-01-01/deploymentTemplate.json [hunk 3]
```diff
@@ -1,509 +1,516 @@
...
-  "$schema": "http://json-schema.org/draft-04/schema#",
-  "title": "Template",
-  "description": "An Azure deployment template",
-  "type": "object",
-  "properties": {
-    "$schema": {
-      "type": "string",
-      "description": "JSON schema reference"
-    },
-    "contentVersion": {
```
Comment (wenwu449) [inline schemas/2015-01-01/deploymentTemplate.json]: please revert this part change to keep the file history consistent.

**Comment by sazeesha** [comment]
I will open a new PR with just the changes I need

**Pull request closed without merging**